third_party/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -164,7 +164,9 @@ impl Interpreter {
                     }
                 },
                 ExitCode::Call(fh_idx) => {
-                    let func = resolver.function_from_handle(fh_idx);
+                    let func = resolver
+                        .function_from_handle(fh_idx)
```

Comment:
Interesting. Before this couldn't error and now it can? Is this only for robustness (turning a panic into an error)?